tests/utils/common-utils.mjs

```diff
@@ -0,0 +1,354 @@
+import fs from 'fs-extra';
+import os from 'node:os';
+
+export class Constants {
+  static get cliPromptMsDelay() {
+    return 500;
+  }
+  static get minaGraphQlPort() {
+    return 8080;
+  }
+  static get minaAccountsManagerPort() {
+    return 8181;
+  }
+  static get mockedEndpointsServicePort() {
+    return 8282;
+  }
+  static get skipInteractiveSelectionOptions() {
+    return [false, true];
+  }
+  static get uiTypes() {
+    return ['next', 'svelte', 'nuxt', 'empty', 'none'];
+  }
+  static get exampleTypes() {
+    return ['sudoku', 'tictactoe'];
+  }
+  static get feePayerMgmtTypes() {
+    return ['recover', 'new', 'cached'];
+  }
+  static get feePayerCacheDir() {
+    return `${os.homedir()}/.cache/zkapp-cli/keys`;
+  }
+  static get feePayerTmpCacheDir() {
+    return `${os.homedir()}/.cache/zkapp-cli/_keys`;
+  }
+  static get recentBlocks() {
+    return 25;
+  }
+  static get specialCliKeys() {
+    return [
+      'arrowDown',
+      'arrowLeft',
+      'arrowRight',
+      'arrowUp',
+      'backSpace',
+      'delete',
+      'end',
+      'enter',
+      'escape',
+      'home',
+      'pageUp',
+      'pageDown',
+      'space',
+      'tab',
+      'ctrlc',
+    ];
+  }
+  static get defaultProjectConfig() {
+    return {
+      version: 1,
+      deployAliases: {},
+    };
+  }
+  static get syncStatusGraphQlResponse() {
+    return {
+      data: {
+        syncStatus: 'SYNCED',
+      },
+    };
+  }
+  static get nonceFetchingGraphQlResponse() {
+    return {
+      data: {
+        account: {
+          nonce: '999',
+        },
+      },
+    };
+  }
+  static getAccountDetailsFetchingGraphQlResponse(publicKey) {
+    return {
+      data: {
+        account: {
+          publicKey,
+          token: 'wSHV2S4qX9jFsLjQo8r1BsMLH2ZRKsZx6EJd1sbozGPieEC4Jf',
```

Comment:
Thanks, the resulting object actually has too many redundant properties that can be removed. It will be simplified to only have properties `zkapp-cli` actually expects.

Reply:
Still needs to be done: https://github.com/o1-labs/quality-and-testing/issues/131